source/administration/plugins.rst

```diff
@@ -4,148 +4,96 @@ Plugins (Beta)
- - **Server plugins (Release Candidate)**: Makes it easier to integrate with third-party systems such as JIRA, GitLab or Jenkins. A sample plugin for a video and audio call system with Zoom `is available here <https://github.com/mattermost/mattermost-plugin-zoom>`__. A sample plugin that auto-censors restricted words and intercepts them `is described here <https://forum.mattermost.org/t/coming-soon-apiv4-mattermost-post-intercept/4982>`__.
+ - **Webapp plugins (Beta)**: Customize the Mattermost user interface by adding to the channel header, overriding the `RHS`_, or even rendering a custom post type. All this is possible without having to fork the source code and rebase on every Mattermost release. For a sample plugin, see `our Zoom plugin <https://github.com/mattermost/mattermost-plugin-zoom>`__.
+ - **Server plugins (Release Candidate)**: Run a Go process alongside the server, filtering messages or integrating with third-party systems such as Jira, GitLab or Jenkins. For a sample plugin, see `our Jira plugin <https://github.com/mattermost/mattermost-plugin-jira>`__.
 
-Later in upcoming releases, the `Enterprise Edition <https://about.mattermost.com/pricing>`__ will also support plugins that enable you to extend Mattermost functionality to meet a specific, complex requirement such as profiling performance metrics, and to implement highly customized compliance rules for `information barriers <http://www.17a-4.com/supervision-information-barriers/>`__.
-
-For a list of available plugins, including those for auto-linking, auto-censoring and anti-virus scanning, as well as third-party plugins with JIRA, Zoom, GitHub and more, see the `Mattermost plugins repo <https://github.com/mattermost/mattermost-plugins>`__. 
+Browse available plugins at `https://integrations.mattermost.com/ <https://integrations.mattermost.com/>`__.
 
 Security
 --------
-Plugins are powerful. You should only install plugins that you've thoroughly reviewed as they have the potential to compromise the security of your installation.
+Plugins are intentionally powerful and not artificially sandboxed in any way. Server plugins can execute arbitrary code alongside your server. Webapp plugins can deploy arbitrary code in client browsers. Plugins effectively become part of the Mattermost server.
 
-Server Considerations
-~~~~~~~~~~~~~~~~~~~~~
-Plugins have the ability to execute arbitrary code on your server. They can do just about *anything*. For example, they could read your config file to get your database password, connect to your database, then exfiltrate sensitive user information.
+While this power enables deep customization and integration, it can be abused in the wrong hands. Plugins have full access to your server configuration and thus your Mattermost database. Plugins can read any message in any channel, or perform any action on behalf of any user in the webapp.
```

Comment:
```suggestion
While this power enables deep customization and integration, it can be abused in the wrong hands. Plugins have full access to your server configuration and thus also to your Mattermost database. Plugins can read any message in any channel, or perform any action on behalf of any user in the webapp.
```